exposee-ui/src/components/Mainpage/Main.css

```diff
@@ -0,0 +1,62 @@
+.main {
+    max-width: 800px;
+    margin: auto;
+    padding: 20px;
+  }
+  
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 20px;
+  }
+  
+  .user-info {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+  
+  .user-info a {
+    margin-top: 10px;
+    margin-right: 20px;
+  }
+  
+  .user-info button {
+    background: none;
```

Comment:
the bg is none by default, not sure you need the background and border set to none